apps/dashboard/app/controllers/projects_controller.rb

```diff
@@ -110,7 +110,45 @@ def job_details
 
     hpc_job = project.job(job_details_params[:jobid].to_s, cluster_str)
 
-    render(partial: 'job_details', locals: { job: hpc_job })
+    @project = project
+
+    render(partial: 'job_details', locals: { job: hpc_job, project: @project })
+  end
+
+  # DELETE /projects/:project_id/jobs/:cluster/:jobid
+  def delete_job
+    @project = Project.find(job_details_params[:project_id])
+
+    cluster_str = job_details_params[:cluster].to_s
+
+    @project.remove_logged_job(job_details_params[:jobid].to_s, cluster_str)
+
+    @valid_project = Launcher.clusters?
+    @valid_scripts = Launcher.scripts?(@project.directory)
+    @scripts = Launcher.all(@project.directory)
+
+    render :show
```

Comment:
I think we should follow a pattern here where `remove_logged_job` returns `true` or `false` based on whether the operation was successful. Then the controller toggles based off of that.

Like say how `update`, `destroy`, `create` or works in this same controller. I think redirecting here is fine (so that we don't have to create/recreate those attributes), it just seems like the toggle is redirect with a notice when successful or an alert for a failure.